pkg/execute/kubectl_cmd_builder_msg.go

```diff
@@ -14,8 +14,31 @@ type (
 	}
 	// KubectlCmdBuilderOption defines option mutator signature.
 	KubectlCmdBuilderOption func(options *KubectlCmdBuilderOptions)
+
+	// KVItem describes the data for the dropdown item.
+	KVItem struct {
```

Comment:
Just a minor thing, `KVItem` seems general  purpose KV storage data structure but it seems this is specifically developed for dropdown items. It is also opened to other packages. Maybe we could use more specific name like `DropdownItem`, just an idea :)

Reply:
good point 👍  I will change to `dropdownItem` so it will be more specific and private